data/resources/resources.gresource.xml

```diff
@@ -5,10 +5,10 @@
     <file preprocess="xml-stripblanks">icons/scalable/actions/clear-symbolic.svg</file>
     <file preprocess="xml-stripblanks">icons/scalable/actions/done-symbolic.svg</file>
     <file preprocess="xml-stripblanks">icons/scalable/actions/edit-symbolic.svg</file>
+    <file preprocess="xml-stripblanks">icons/scalable/status/background-app-ghost-symbolic.svg</file>
     <file preprocess="xml-stripblanks">icons/scalable/status/message-failed-symbolic.svg</file>
     <file preprocess="xml-stripblanks">icons/scalable/status/message-pending-symbolic.svg</file>
     <file preprocess="xml-stripblanks">icons/scalable/status/message-read-symbolic.svg</file>
-    <file preprocess="xml-stripblanks">icons/scalable/status/message-unread-left-symbolic.svg</file>
```

Comment:
Not sure why you removed this.

![grafik](https://github.com/paper-plane-developers/paper-plane/assets/3630213/25ebf78e-3913-4f2b-a3b2-68d2a823870b)

Reply:
looks like I cut a line instead of copying it

Reply:
I have no idea, but it's fixed now